packages/debug-ui/gulpfile.js

```diff
@@ -67,6 +67,7 @@ tsPipeline.registerBuildGulpTasks(
             index: __dirname + '/' + config.srcDir + 'code/indexEntry.ts',
             newsroom: __dirname + '/' + config.srcDir + 'code/newsroomEntry.ts',
             article: __dirname + '/' + config.srcDir + 'code/articleEntry.ts',
+            base: __dirname + '/' + config.srcDir + 'code/base.ts',
```

Comment:
What's the difference between base and index?

Reply:
Also, please don't use base.pug as an entry, it should be treated as an abstract class to be extended by other entries